types/highland/index.d.ts

```diff
@@ -766,6 +766,26 @@ declare namespace Highland {
 		pluck<U>(prop: string): Stream<U>;
 
 		/**
+		 * Limits number of values through the stream to a maximum of number of values
+		 * per window. Errors are not limited but allowed to pass through as soon as
+		 * they are read from the source.
+		 *
+		 * @id ratelimit
+		 * @section Transforms
+		 * @name Stream.ratelimit(num, ms)
+		 * @param {Number} num - the number of operations to perform per window
+		 * @param {Number} ms - the window of time to limit the operations in (in ms)
+		 * @api public
+		 *
+		 * _([1, 2, 3, 4, 5]).ratelimit(2, 100);
+		 *
+		 * // after 0ms => 1, 2
+		 * // after 100ms => 1, 2, 3, 4
+		 * // after 200ms => 1, 2, 3, 4, 5
+		 */
+		ratelimit<U>(num: number, ms: number): Stream<U>;
```

Comment:
I think `ratelimit` does not change the type of the elements of the stream, so the return type should be `Stream<R>` shouldn't it?

Reply:
Good catch! I've just amended that in 615e801485ffb699f4fd3cf9f66f3c644bab8c86